react/components/Popper.tsx

```diff
@@ -171,17 +170,15 @@ const Popper = forwardRef(function Popper(props: Props, ref) {
   }
 
   return (
-    <Portal>
-      <div
-        role={role}
-        ref={handleRef}
-        style={{ position: 'fixed', left: 0, top: 0, ...style }}
-        className={`${styles.popper} ${handles.popper}`}
-        {...rest}
-      >
-        {typeof children === 'function' ? children(childProps) : children}
-      </div>
-    </Portal>
+    <div
+      role={role}
+      ref={handleRef}
+      style={{ position: 'fixed', left: 0, top: 0, ...style }}
+      className={`${styles.popper} ${handles.popper}`}
```

Comment:
```suggestion
      style={style}
      className={`${styles.popper} ${handles.popper} fixed left-0 top-0`}
```

Reply:
I did this way because these styles if you remove it something really strange happens with the `popperjs` and nothing will be shown, so it makes sense to me these styles be applied directly